Makefile

```diff
@@ -1,4 +1,7 @@
 
+# Default is sh, which has scripting limitations
+SHELL := $(shell command -v bash;)
```

Comment:
Needed for timebomb-check below. I looked through the Makefile, cannot find any place where this should make a difference, but it's still worth close attention.

Reply:
Not sure I follow how this is related to timebomb, but I trust you if you say so.  In any case this is a good idea.  That `sh` assumption is easy to make.

Reply:
sh will not handle `while ... done < <(command)`

Reply:
Ahh ok, gotcha.